platform/o.n.swing.plaf/src/org/netbeans/swing/plaf/util/UIUtils.java

```diff
@@ -25,21 +25,12 @@
 import java.awt.*;
 import java.lang.reflect.Method;
 import java.net.URL;
-import java.util.HashMap;
-import java.util.Map;
 
 /** XP color scheme installer.
  *
  * @author  Dafe Simonek
  */
 public final class UIUtils {
-    private static HashMap<RenderingHints.Key, Object> hintsMap = null;
-    private static final boolean noAntialias =
-        Boolean.getBoolean("nb.no.antialias"); //NOI18N
```

Comment:
This property will no longer be respected. I think that's fine; it doesn't seem official, or needed anymore.